README.md

```
@@ -58,16 +64,21 @@ The folder structure follows a few conventions and ideas. The order of the items
 enumerated below is the order how they are currently enumerated within the primary
 navigation element (left-hand menu).
 
+**Learning path**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)